CHANGELOG.rst

```diff
@@ -73,6 +73,8 @@ Changelog
   :class:`~cryptography.hazmat.primitives.ciphers.aead.AESSIV`, and
   :class:`~cryptography.hazmat.primitives.ciphers.aead.ChaCha20Poly1305` to
   allow encrypting directly into a pre-allocated buffer.
+* Added support for PKCS1v15 signing without DigestInfo using
+  :class:~cryptography.path.to.NoDigestInfo.
```

Comment:
This was just intended as an example. You need to define the actual path to NoDigestInfo here.